### PR TITLE
Update Microsoft.WindowsAppSDK to 1.6 stable

### PR DESCRIPTION
**Pull request by Arlodotexe**
This PR updates our tooling to use the recent Microsoft.WindowsAppSDK 1.6 stable release.
 
Progress towards https://github.com/CommunityToolkit/Tooling-Windows-Submodule/issues/205

### CODE REVIEW

**Review by michael-hawker — APPROVED** [review]

File: MultiTarget/PackageReferences/WinAppSdk.props [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <Project>
   <ItemGroup>
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.240821007-preview2" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.240829007" />
     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
     <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2730-prerelease" />
```
Comment (michael-hawker) [inline MultiTarget/PackageReferences/WinAppSdk.props:5]: @Arlodotexe we should try removing this now, eh? As this was temporary with one of the previews right?

**Review by michael-hawker — CHANGES_REQUESTED** [review]
Think this is the discussion point we've been miscommunicating on, eh? (Sorry, there's been a lot of things going on as of late!)

File: MultiTarget/PackageReferences/WinAppSdk.props (hunk 1 above)
Comment (Arlodotexe) [reply MultiTarget/PackageReferences/WinAppSdk.props]: Right, this was a workaround for a problem reported during the update to preview1 [here](https://github.com/CommunityToolkit/Labs-Windows/pull/561#issuecomment-2287775062), and using the prerelease package is the recommended solution from @Sergio0694 [here](https://github.com/CommunityToolkit/Labs-Windows/pull/561#issuecomment-2287775062). 

If all is stable with 1.6, this shouldn't be needed anymore. We can remove it and let it fall back to the version included in 1.6 stable.

File: MultiTarget/PackageReferences/WinAppSdk.props [hunk 2]
```diff
@@ -1,8 +1,7 @@
 <Project>
   <ItemGroup>
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.240821007-preview2" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.240829007" />
     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
-    <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2730-prerelease" />
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Windows.CsWinRT" Version="2.1.1" />
```
Comment (michael-hawker) [inline MultiTarget/PackageReferences/WinAppSdk.props:7]: We may want a note here that we may not need this explicit reference once the update to the .NET SDK goes out (per the 1.6 release notes)

Comment (Arlodotexe) [reply MultiTarget/PackageReferences/WinAppSdk.props:7]: Seems like it's building fine without that reference now, see https://github.com/CommunityToolkit/Tooling-Windows-Submodule/pull/209#discussion_r1750944230.

**Pull request merged**